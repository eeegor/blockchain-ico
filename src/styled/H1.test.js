import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import { H1 } from '.';

describe('H1', () => {
	test('renders default h1', () => {
		const tree = renderer.create(<H1>Title</H1>).toJSON();
		expect(tree).toMatchSnapshot();
	});

	test('renders h1 with custom color', () => {
		const tree = renderer.create(<H1 color="green">Title</H1>).toJSON();
		expect(tree).toMatchSnapshot();
		expect(tree).toHaveStyleRule('color', 'green');
	});
});
