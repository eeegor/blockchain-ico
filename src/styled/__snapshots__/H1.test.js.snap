// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H1 renders default h1 1`] = `
.c0 {
  font-size: 34px;
  line-height: 51px;
  font-family: Rubik-Regular;
  margin-bottom: 12px;
}

<h1
  className="c0"
>
  Title
</h1>
`;

exports[`H1 renders h1 with custom color 1`] = `
.c0 {
  font-size: 34px;
  line-height: 51px;
  font-family: Rubik-Regular;
  margin-bottom: 12px;
  color: green;
}

<h1
  className="c0"
  color="green"
>
  Title
</h1>
`;
